x-pack/legacy/plugins/siem/public/pages/hosts/details/helpers.test.ts

```diff
@@ -35,4 +35,32 @@ describe('hosts page helpers', () => {
       ).toEqual('');
     });
   });
+
+  describe('getHostDetailsPageFilters', () => {
+    it('correctly constructs pageFilters for the given hostName', () => {
+      expect(getHostDetailsPageFilters('host-1')).toEqual([
+        {
+          meta: {
+            alias: null,
+            negate: false,
+            disabled: false,
+            type: 'phrase',
+            key: 'host.name',
+            value: 'host-1',
+            params: {
+              query: 'host-1',
+            },
+          },
+          query: {
+            match: {
+              'host.name': {
+                query: 'host-1',
+                type: 'phrase',
+              },
+            },
+          },
+        },
+      ]);
+    });
+  });
```

Comment:
👍 Thanks for the test!

Reply:
Optionally you can do this trick:

```ts
    it('correctly constructs pageFilters for the given hostName', () => {
      const expected: esFilters.Filter[] = [
        {
          meta: {
            alias: null,
            negate: false,
            disabled: false,
            type: 'phrase',
            key: 'host.name',
            value: 'host-1',
            params: {
              query: 'host-1',
            },
          },
          query: {
            match: {
              'host.name': {
                query: 'host-1',
                type: 'phrase',
              },
            },
          },
        },
      ];
      expect(getHostDetailsPageFilters('host-1')).toEqual(expected);
```

And if that type ever changes your type script will blow up before the test does and ensure that your expected type here always matches your filter type.

Reply:
Ooh nice little improvement there -- thanks @FrankHassanabad! I've made the change and will take this little tidbit with me going forward 🙂